code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -286,7 +287,8 @@
 		animation_number++
 		if(!is_servant_of_ratvar(L))
 			var/vitality_drained = 0
-			if(L.stat == DEAD)
+			if(L.stat == DEAD && !consumed_vitality)
+				consumed_vitality = TRUE //Prevent the target from being consumed multiple times
```

Comment:
Shouldn't this be unset somewhere ?

Reply:
it's unnecessary if the mob is being qdeleted, you just need to change the while(L to while(!QDELETED(L)

Reply:
I tried that initially, and it seems that it doesn't pick it up because `dust()` uses a `QDEL_IN` timer.

Reply:
but you were calling qdel on the mob explicitly before the last commit. you should remove this, since you don't need it if you're outright calling qdel, and make the check for L a !QDELETED() check.

Reply:
I don't really follow. Now it uses the standard `dust()` call instead of calling `qdel`, and using a `QDELETED` check for a mob being dusted didn't work when I tried it.

Reply:
it didn't work to check if it was being dusted because you weren't calling qdel at the time.

Reply:
I'm not here either.

Reply:
This is fine, ideally dust needs to be reworked.